pkg/snapshot/velero.go

```diff
@@ -379,6 +393,51 @@ func getVersion(nameSpace string) (string, error) {
 	return serverStatus.Status.ServerVersion, nil
 }
 
+func getVeleroPod(ctx context.Context, clientset *kubernetes.Clientset, namespace string) (string, error) {
+	veleroLabels := map[string]string{
+		"component": "velero",
+		"deploy":    "velero",
+	}
+
+	veleroPods, err := clientset.CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: labels.SelectorFromSet(veleroLabels).String(),
+	})
+	if err != nil {
+		return "", errors.Wrap(err, "failed to list velero pods before restarting")
+	}
+
+	for _, pod := range veleroPods.Items {
+		if pod.Status.Phase == corev1.PodRunning {
+			return pod.Name, nil
+		}
+	}
+
+	return "", errors.New("unable to find velero pod")
```

Comment:
I think this needs some consistency.  If `getVeleroPod` finds 0 pods, it will fail.  But if `getResticPods` finds 0 pods, it will succeed.